boards/holybro/kakuteh7mini/firmware.prototype

```diff
@@ -1,10 +1,10 @@
 {
-    "board_id": 1054,
+    "board_id": 1058,
```

Comment:
It's a bit weird to "change" this like that.

What about existing boards? They suddenly have a different board ID between bootloader and firmware.

It's also confusing to have the same board with the same name twice in this list:
https://github.com/PX4/PX4-Bootloader/blob/e40999bfe9d6ea5da0dd717f5eb09adbd1f1c7d1/board_types.txt#L167C1-L171

Reply:
Did something change regarding NAND?
Should we add a new board ID?